connector/src/main/resources/config.yml

```diff
@@ -89,15 +89,15 @@ command-suggestions: true
 
 # The following three options enable "ping passthrough" - the MOTD, player count and/or protocol name gets retrieved from the Java server.
 # Relay the MOTD from the remote server to Bedrock players.
-passthrough-motd: false
+passthrough-motd: true
 # Relay the protocol name (e.g. BungeeCord [X.X], Paper 1.X) - only really useful when using a custom protocol name!
 # This will also show up on sites like MCSrvStatus. <mcsrvstat.us>
-passthrough-protocol-name: false
+passthrough-protocol-name: true
 # Relay the player count and max players from the remote server to Bedrock players.
-passthrough-player-counts: false
+passthrough-player-counts: true
 # Enable LEGACY ping passthrough. There is no need to enable this unless your MOTD or player count does not appear properly.
 # This option does nothing on standalone.
-legacy-ping-passthrough: false
+legacy-ping-passthrough: true
```

Comment:
> There is no need to enable this unless your MOTD or player count does not appear properly.

Reply:
In case someone does need to enable it, they don't have to mess with the configs. It does not make a major performance change, so why not?

Reply:
I'm neutral about the first 3 options, but regarding the legacy-ping-passthrough:

It's not an option that is commonly used because it isn't commonly necessary. If I understand correctly:

1. it technically isn't as performant
2. You're also more likely to get errors in the console due to it

Reply:
Fair Point. I'm fairly new to git, so is there a way I can edit the PR without making a new one?

Reply:
Yeah, I'm against this change, if only because it has the chance to break things like BungeeCord that can time out IP addresses if they ping too much.